src/erdstall.ts

```diff
@@ -105,7 +106,9 @@ export interface ErdstallClient
 		Initializer,
 		Subscriber,
 		NFTMetadataProvider,
-		AccountGetter {}
+		AccountGetter {
+	readonly tokenCache: TokenCache;
```

Comment:
If we rename `TokenCache` to `TokenProvider`, I'd also rename this then to `tokenProvider`.